docs/examples/notebooks/risk_categorization.ipynb

```diff
@@ -6,10 +6,12 @@
    "metadata": {},
    "source": [
     "## Risk Categorization\n",
-    "\n",
     "Categorize the severity of risks linked to an AI system usecase, which includes the domain, purpose, capabilities, AI user, and AI subject.\n",
     "\n",
-    "#### Load libaries.\n"
+    "### Attribution\n",
+    "The author of this notebook is Dhaval Salwala (IBM Research) and the underlying risk classification methodology according to the EU AI Act was developed by the Social and Responsible AI Team at Nokia Bell Labs, Cambridge UK (ExploreGen paper link contributors: Viviane Herdel, Sanja Šćepanović, Edyta Bogucka, Daniele Quercia).   \n",
+    "\n",
```

Comment:
Hi @ingelise 
This looks fine. Should we add the paper citation as well in the cell ?

```
Herdel, Viviane, Sanja Šćepanović, Edyta Bogucka, and Daniele Quercia. "ExploreGen:
Large language models for envisioning the uses and risks of AI technologies." In Proceedings
of the AAAI/ACM Conference on AI, Ethics, and Society, vol. 7, pp. 584-596. 2024.
```